simulariumio/cytosim/cytosim_converter.py

```diff
@@ -235,13 +236,20 @@ def _parse_objects(
                 result.subpoints[time_index][agent_index][
                     subpoint_index : subpoint_index
                     + SUBPOINT_VALUES_PER_ITEM(DISPLAY_TYPE.FIBER)
-                ] = scale_factor * np.array(
+                ] = np.array(
                     [
                         float(columns[1].strip("+,")),
                         float(columns[2].strip("+,")),
                         float(columns[3].strip("+,")),
                     ]
                 )
+                TrajectoryConverter.check_max_min_coordinates(
+                    max_dimensions,
+                    min_dimensions,
+                    result.subpoints[time_index][agent_index][
+                        subpoint_index : subpoint_index + 3
+                    ],
+                )
```

Comment:
Instead of doing this each time we read an XYZ position, we can do it after we finish going through and getting all the positions using numpy slicing. Let's talk about it in 1:1